target/linux/ramips/image/mt7620.mk

```diff
@@ -244,6 +244,15 @@ define Device/u25awf-h1
 endef
 TARGET_DEVICES += u25awf-h1
 
+define Device/u35wf
+  DTS := U35WF
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := Kimax U35WF
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-mt76 kmod-usb-ohci \
```

Comment:
no usb LED, no need for `kmod-usb-ledtrig-usbport`. I'm not sure about the other packages. Usually we only add `kmod-usb2 kmod-usb-ohci` for usb. But considering the purpose of the board the other packages kind of make sense.

Reply:
reviewed, thanks